server/src/controllers/Events/resolver.ts

```diff
@@ -719,6 +719,7 @@ ${venueDetails}`;
     return event;
   }
 
+  @Authorized(Permission.EventEdit)
```

Comment:
I think it would be better to add separate permission for deleting event, instead of using the same one as for editing it.